.circleci/config.yml

```diff
@@ -98,7 +98,7 @@ workflows:
   version: 2
   test:
     jobs:
-      - test_high_sierra
+      # - test_high_sierra
```

Comment:
this is only used in my fork, but high sierra is flaking a lot, so turning off for now.